sdk/python/feast/infra/offline_stores/contrib/mssql_offline_store/tests/data_source.py

```diff
@@ -0,0 +1,105 @@
+from typing import Dict, List
+
+import pandas as pd
+import pytest
+from sqlalchemy import create_engine
+from testcontainers.core.waiting_utils import wait_for_logs
+from testcontainers.mssql import SqlServerContainer
+
+from feast.data_source import DataSource
+from feast.infra.offline_stores.contrib.mssql_offline_store.mssql import (
+    MsSqlServerOfflineStoreConfig,
+    _df_to_create_table_sql,
+)
+from feast.infra.offline_stores.contrib.mssql_offline_store.mssqlserver_source import (
+    MsSqlServerSource,
+)
+from feast.saved_dataset import SavedDatasetStorage
+from tests.integration.feature_repos.universal.data_source_creator import (
+    DataSourceCreator,
+)
+
+MSSQL_USER = "SA"
+MSSQL_PASSWORD = "yourStrong(!)Password"
+
+
+@pytest.fixture(scope="session")
+def mssql_container():
+    container = SqlServerContainer(user=MSSQL_USER, password=MSSQL_PASSWORD)
```

Comment:
pass in the azure-sql-edge image as mentioned above?

Reply:
Oh I didn't realize you actually wanted to use this image. Updated.